src/lib/device-online-state.ts

```diff
@@ -160,45 +210,76 @@ class DeviceOnlineStateManager {
 					throw err;
 				}
 			})
-			.then(() => {
-				return this.consume();
+			.then(() =>
+				this.setupQueueStatsEmitter(
+					DeviceOnlineStateManager.QUEUE_STATS_INTERVAL_MSEC,
+				),
+			);
+	}
+
+	private async setupQueueStatsEmitter(interval: number) {
+		return setTimeout(async () => {
+			const startAt = Date.now();
+			const queueAttributes = await this.rsmq.getQueueAttributesAsync({
```

Comment:
Can this fail? It's probably worth adding a try/catch with a captureException to be safe in that case, and put the `this.setupQueueStatsEmitter(interval)` in a `finally`